js/src/collapse.js

```diff
@@ -72,11 +72,14 @@ const Collapse = (($) => {
         `[data-toggle="collapse"][href="#${element.id}"],` +
         `[data-toggle="collapse"][data-target="#${element.id}"]`
       ))
-      const tabToggles = $(Selector.DATA_TOGGLE)
-      for (let i = 0; i < tabToggles.length; i++) {
-        const elem = tabToggles[i]
+      const toggleList = [].slice.call(document.querySelectorAll(Selector.DATA_TOGGLE))
+      for (let i = 0; i < toggleList.length; i++) {
```

Comment:
Maybe cache the length in the loops

Reply:
it was interesting in old browsers but not anymore as far as I know

Reply:
Doesn't hurt to do it either.

Reply:
in that case I'll target that in an other PR, to fix all at once 😉